src/main/scala/com/softwaremill/mqperf/mq/ArtemisMq.scala

```diff
@@ -0,0 +1,18 @@
+package com.softwaremill.mqperf.mq
+
+import javax.jms.ConnectionFactory
+
+import com.softwaremill.mqperf.config.TestConfig
+import org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory
+
+class ArtemisMq(val testConfig: TestConfig) extends Jms2Mq {
+
+  override lazy val connectionFactory: ConnectionFactory = {
+    val hosts = "failover:(" + testConfig.brokerHosts.map(h => s"tcp://$h:61616").mkString(",") + ")"
+    val cf = new ActiveMQConnectionFactory(hosts)
+    cf.setConfirmationWindowSize(10485760)
+    cf.setBlockOnDurableSend(false)
```

Comment:
@michaelandrepearce Thanks, so how would `send+CompletionListener+blockOnDurableSend=true` behave?

As for the tuning, I'm fine with changing some basic settings, but on the other hand I don't want to create finely tuned settings only for this benchmark, as this might depart quite far from what people might actually use. That's the same for all of the queues tested. That's also why I'm sticking with what can be found in the documentation.